bundles/org.openhab.binding.velux/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 	*
+	* Copyright (c) 2010-2019 Contributors to the openHAB project 
+	*  
+	*  See the NOTICE file(s) distributed with this work for additional 
+	*  information. 
+	*  
+	*  This program and the accompanying materials are made available under the 
+	*  terms of the Eclipse Public License 2.0 which is available at 
+	*  http://www.eclipse.org/legal/epl-2.0 
+	*  
+	*  SPDX-License-Identifier: EPL-2.0 
+	*  
+	*  @author Guenther Schreiner - Initial contribution. 
+	*  @author Andrew Fiddian-Green - minor corrections. 
+	*  @since 2.4.0 
+	*
+ -->
+<!-- Velux Binding -->
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+	https://openhab.org/schemas/config-description-1.0.0.xsd">
+	<!-- -->
+	<!-- Velux Bridge KLF200 -->
+	<!-- -->
+	<config-description uri="bridge-type:velux:bridge">
+		<parameter name="bridgeIPAddress" type="text">
```

Comment:
Changed.

Reply:
Do you think similar replacements should be done for:

- protocol instead of bridgeProtocol,
- tcpPort instead of bridgeTCPPort,
- password instead of bridgePassword?

Reply:
> Do you think similar replacements should be done for:
> 
> protocol instead of bridgeProtocol,
> tcpPort instead of bridgeTCPPort,
> password instead of bridgePassword?
> 
Personally I think yes.

Reply:
Yes I intended it for the other parameters as well :smile: